mobile/src/libs/components/modal/modal.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Modal as RNModal } from 'react-native';
+
+import { Icon, Pressable, View } from '#libs/components/components';
+import { AppColor } from '#libs/enums/enums';
+
+import { styles } from './styles';
+
+type Properties = {
+  isVisible: boolean;
+  onClose: () => void;
+  onDelete?: () => void;
+  type?: 'Note';
+  children?: React.ReactNode;
+};
+
+const Modal: React.FC<Properties> = ({
+  children,
+  isVisible,
+  onClose,
+  onDelete,
+}) => {
+  return (
+    <RNModal visible={isVisible} transparent={true} animationType="none">
+      <Pressable onPress={onClose} style={styles.background}>
+        <View style={styles.modalContainer}>
+          <Pressable>
+            <View style={styles.modalView}>
+              {onDelete ? (
+                <></>
```

Comment:
could you please add full code for the modal as i have sent to you ? because im aftaid that this empty conditional can overwrite modal if it will be merged later

Reply:
Honestly, I don't see any issue here. We're bound to have conflicts, and either you or I (depending on whose PR gets merged first) can handle everything properly